src/Codeception/TestCase/Cest.php

```diff
@@ -212,4 +213,16 @@ protected function executeHook($I, $hook)
         }
     }
 
+    /**
+     * @param string $key
+     * @return array|string
+     */
+    public function getParameter($key)
```

Comment:
actually this is uselss. You are not accessing Cest class from inside your tests in any way